features/user-reset-password.feature

```diff
@@ -41,3 +41,23 @@ Feature: Reset passwords for one or more WordPress users.
       """
       {ORIGINAL_PASSWORD}
       """
+
+@require-wp-4.3
+Scenario: Reset the password of a WordPress user, and show the new password
+  Given a WP installation
+
+  When I run `wp user get 1 --field=user_pass`
+  Then save STDOUT as {ORIGINAL_PASSWORD}
+
+  When I run `wp user reset-password 1 --skip-email --show-password`
```

Comment:
This absolutely should capture the new password, and the next step should check for it, but I have no idea how to do that :(

Reply:
... I have no idea what I was hung up on, disregard :)